src/App.js

```diff
@@ -39,6 +40,11 @@ firebase.initializeApp(firebaseConfig);
 export default function App() {
   return (
     <div className="main">
+      <Helmet>
```

Comment:
I see your point, I'll remove react helmet and put it in the index.html file. I just saw some online suggestions for React Helmet and thought it would be an interesting dependency to add. Thanks for pointing that out.